pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1311,3 +1311,17 @@ def test_dropna(fill_value):
     df = pd.DataFrame({"a": [0, 1], "b": arr})
     expected_df = pd.DataFrame({"a": [1], "b": exp}, index=pd.Int64Index([1]))
     tm.assert_equal(df.dropna(), expected_df)
+
+
+def test_maxmin():
+    data = np.arange(10).astype(float)
```

Comment:
can you make sure some NaNs are in this, and assert that we are correctly hitting those (can parameterize to do that)

Reply:
also a tests on all-NaN